website/markdown/docs/building-at-scale/caching.mdx

```diff
@@ -45,6 +51,10 @@ tuist cache warm
 
 One the cache is warmed, you can use the [focus](/docs/commands/focus/) command to generate a project replacing direct and transitive dependencies with artifacts from the cache.
 
+### Caching profile 
+
+The caching profile allows users to specify how targets will be cached, for example setting a configuration. Instead of passing the configuration through CLI arguments, developers can define a profile and reference it with `tuist cache warm --profile MyProfile`.
```

Comment:
As I wrote to you on Slack, I'd give also a quick example of how could a `Profile`, eg:

You can define a Profile in `Config.swift`, for example like this:
```swift
let config = Config(cache: .cache(profiles: [.profile(name: "Simulator", configuration: "Debug",)])
```

Reply:
Sure, I updated the documentation 👍